slamd/materials/processing/models/aggregates.py

```diff
@@ -1,18 +1,15 @@
+from dataclasses import dataclass
 from slamd.materials.processing.models.material import Material
 
 
-class Aggregates(Material):
-
-    def __init__(self, name='', type='', costs=None, composition=None, additional_properties=None):
-        super().__init__(name=name, type=type, costs=costs,
-                         additional_properties=additional_properties)
-        self.composition = composition
-
-
+@dataclass
 class Composition:
+    fine_aggregates: float = None
+    coarse_aggregates: float = None
+    fa_density: str = None
+    ca_density: str = None
+
 
-    def __init__(self, fine_aggregates=None, coarse_aggregates=None, fa_density=None, ca_density=None):
-        self.fine_aggregates = fine_aggregates
-        self.coarse_aggregates = coarse_aggregates
-        self.fa_density = fa_density
-        self.ca_density = ca_density
+@dataclass
+class Aggregates(Material):
+    composition: Composition = None
```

Comment:
Using a dataclass decorator for a class which inherits from another one, does it also get the instance variables from the parent?